kuma/static/styles/minimalist/components/_main-nav.scss

```diff
@@ -4,19 +4,32 @@
 [mdn-static] ~ main navigation menu
 ********************************************************************** */
 .main-nav {
-    grid-area: M;
     @include set-font-size(18px);
+}
 
-    & > ul {
-        display: block;
+.main-menu {
+    display: none;
+    position: absolute;
+    top: 80px;
+    left: 0;
+    background-color: #fff;
+    padding: 20px;
+    width: 100%;
+    height: 100vh;
+    font-weight: bold;
+    z-index: $menu-zindex;
+
```

Comment:
Can you add a `box-sizing: border-box` so we don't have a horizontal scroll caused by the padding and 100% width?